src/babashka/nrepl/impl/server.clj

```diff
@@ -14,13 +14,17 @@
 
 (set! *warn-on-reflection* true)
 
+(def pretty-print-fns
+  ['clojure.core/prn 'clojure.core/pprint])
```

Comment:
Turn this in a map from symbol to function:

```
{'clojure.core/prn prn
 'clojure.pprint/pprint clojure.pprint/pprint}
```

The reason is that we want to avoid `resolve` since that doesn't work well with GraalVM.